packages/cohort/src/replicator/services/replicator_service.rs

```diff
@@ -20,6 +20,7 @@ where
     M: MessageReciever<Message = ChannelMessage> + Send + Sync,
 {
     info!("Starting Replicator Service.... ");
+    let mut interval = tokio::time::interval(Duration::from_millis(10_000));
```

Comment:
Consider extracting into configuration property

Reply:
I will update this in the new PR I am working on parallel installs, as I need to add few more environment variables in that PR. Will do it in one go.